src/app/terms/page.js

```diff
@@ -0,0 +1,94 @@
+"use client";
+/* eslint-disable react/no-unescaped-entities */
+
+import { Box, Typography } from '@mui/material';
+
+
+
+export default function TermsAndConditionsPage() {
+  return (
+    <Box sx={{
+      fontFamily: 'Manrope, sans-serif',
+      backgroundColor: '#F2F2F2',
+      p: '40px 20px',
+      minHeight: '100vh',
+      display: 'flex',
+      flexDirection: 'column',
+      alignItems: 'center',
+      }}>
+      <Box sx={{
+        width: '1117.73px',
+        height: '413.25px',
+        backgroundColor: '#E9ECEF',
+        borderRadius: '8.65849px',
+        p: '20px',
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'center',
+        alignItems: 'center',
+        marginTop: '100px', // Adjust this value to reduce the space
+        }}>
+        <Typography variant="h4" sx={{ fontWeight: 'bold' }}>
+          Terms and conditions
+        </Typography>
+        <Typography variant="body1">
+          Last updated: 21 September 2023
+        </Typography>
+      </Box>
+      <Box sx={{ width: 1116.16, p: '20px',fontSize: '16px', lineHeight: '24px' }}>
+        <Typography variant="body1" paragraph>
+          These Terms and Conditions (the "Terms") describe the terms and conditions on which OOSCCA Ltd., a United Kingdom company (“OOSCCA,” “we,” “us,” or “our”) makes its website (“Site”) and application (collectively the “Services”) available to you. By using or accessing our Services, you confirm your agreement to be bound by these Terms and our Privacy Policy, which are incorporated herein by reference. If you do not agree to these Terms and the Privacy Policy, please do not use or access the Services. The Terms expressly supersede any prior terms of use between you and us or any of our affiliates or predecessors.
+        </Typography>
+        <Typography variant="h6" gutterBottom component="div" sx={{ fontWeight: 'bold' }}>
+          Acceptance of the Terms
+        </Typography>
+        <Typography variant="body1" paragraph>
+        OOSCCA's products, features and offerings are available made available through www.OOSCCA.com and app.OOSCCA.com. All of these are collectively referred to as "OOSCCA Properties" or our "Services". The material, including without limitation information, data, text, editorial content, design elements, look and feel, formatting, graphics, images, photographs, videos, music, sounds and other content contained in or delivered via the Services or otherwise made available by OOSCCA in connection with the Services is the "Site Content" (or "Content"). Any material (including the foregoing categories) that you contribute, provide, post or make available using the Services is "Your Content."
```

Comment:
apostroph should be escaped. Do you have ESLint set up?

Reply:
still needs to be fixed